package.json

```diff
@@ -33,7 +33,8 @@
     "ft_test": "mocha --compilers js:babel-core/register --recursive tests/functional",
     "lint": "eslint $(git ls-files '*.js')",
     "lint_md": "mdlint $(git ls-files '*.md')",
-    "start": "node index.js"
+    "start": "node index.js",
+    "test": "mocha --compilers js:babel-core/register --recursive tests/unit"
```

Comment:
I just realized you commented on the rebase of other PR. I am moving this change over there.